GoogleForCreators/web-stories-wp
# Redesign: Video element Play/Stop buttons

## Summary
Updates the buttons of the video element for Playing/Stopping.


## Relevant Technical Choices



## To-do



## User-facing changes
|Before|After|
|------|-----|
| <img width="201" alt="Screenshot 2021-03-22 at 18 41 16" src="https://user-images.githubusercontent.com/3294597/112025802-d354a700-8b35-11eb-979e-2e40d78f59b3.png"> | <img width="209" alt="Screenshot 2021-03-22 at 18 40 33" src="https://user-images.githubusercontent.com/3294597/112025827-dbace200-8b35-11eb-8e5b-e57411c77c50.png"> |



### QA






This PR can be tested by following these steps:

1. Add a video element
2. Select it and hover on it
3. Play and stop the video
4. Verify the icons match [Figma](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories-Post-Stable?node-id=5707%3A454725)

### UAT







This PR can be tested by following these steps:

1.
## Checklist




---



Fixes #6380
